src/EditorFeatures/Core/CodeDefinitionWindow/DefinitionContextTracker.cs

```diff
@@ -206,22 +206,12 @@ private async Task UpdateForCaretPositionAsync(SnapshotPoint pointInRoslynSnapsh
         return ImmutableArray<CodeDefinitionWindowLocation>.Empty;
     }
 
-    private static async Task<ImmutableArray<INavigableItem>> GetDefinitionsAsync(Document document, int position, CancellationToken cancellationToken)
+    private static async Task<ImmutableArray<INavigableItem>> GetNavigableItemsAsync(Document document, int position, CancellationToken cancellationToken)
     {
         // Try IFindDefinitionService first. Until partners implement this, it could fail to find a service, so fall back if it's null.
-        var findDefinitionService = document.GetLanguageService<IFindDefinitionService>();
-        if (findDefinitionService != null)
-            return await findDefinitionService.FindDefinitionsAsync(document, position, cancellationToken).ConfigureAwait(false);
-
-        // Removal of this codepath is tracked by https://github.com/dotnet/roslyn/issues/50391. Once it is removed,
-        // this GetDefinitions method should be inlined into call sites.
-        var goToDefinitionsService = document.GetLanguageService<IGoToDefinitionService>();
-        if (goToDefinitionsService != null)
-        {
-            var result = await goToDefinitionsService.FindDefinitionsAsync(document, position, cancellationToken).ConfigureAwait(false);
-            return result.AsImmutableOrEmpty();
-        }
```

Comment:
removing hte fallback cases.  We now just use the new system all the way through.